### PR TITLE
deploy: Patch the correct images in the correct places

### PR DESCRIPTION
**Pull request by david-caro**
Bug: T316854

### CODE REVIEW

**Comment by david-caro** [comment]
Works locally with both buildpack-admission and registry-admission webhooks enabled, will test on toolsbeta

**Comment by david-caro** [comment]
Now tested in toolsbeta :)

File: deploy/base-tekton/tekton-pipelines-controller-patch.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  {
+    "op": "replace",
+    "path": "/spec/template/spec/containers/0/args/13",
+    "value": "docker-registry.tools.wmflabs.org/toolforge-tektoncd-pipeline-cmd-workingdirinit:v0.33.2"
+  },
+  {
+    "op": "replace",
+    "path": "/spec/template/spec/containers/0/args/17",
+    "value": "docker-registry.tools.wmflabs.org/toolforge-distroless-base@sha256:3cebc059e7e52a4f5a389aa6788ac2b582227d7953933194764ea434f4d70d64"
```
Comment (david-caro) [inline deploy/base-tekton/tekton-pipelines-controller-patch.json:45]: docker-registry.tools.wmflabs.org/toolforge-distroless-base@sha256:eebb155bd1116e3b67e2ce43244f9c9958df0cbb75a84c231565fae2ed87c9f4

**Review by dhinus — APPROVED** [review]
:shipit:

**Pull request merged**